<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwfMp3Looper</class>
 <widget class="QWidget" name="SwfMp3Looper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>669</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Create a looped MP3 in a SWF (v1.1)</string>
  </property>
  <layout class="QHBoxLayout" name="verticalLayoutMainContainer">
   <item>
    <layout class="QVBoxLayout" name="leftLayout">
     <item>
      <widget class="QListView" name="itemList">
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="addRemoveLayout">
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <item>
      <layout class="QHBoxLayout" name="sourceFileLine">
       <item>
        <widget class="QLabel" name="labelSourceFile">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Audio Source File:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sourceFileLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>(Required)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="actionScriptLine">
       <item>
        <widget class="QLabel" name="labelClassName">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>ActionScript 3 Class Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="classNameLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>(Required)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="checkboxAndErrorLine">
       <item>
        <widget class="QCheckBox" name="useDefaultAudioSettings">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Use Default Audio Settings</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="errorLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="audioSettingsGroupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Audio Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutAudioSettings">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutCodec">
          <item>
           <widget class="QLabel" name="labelCodec">
            <property name="text">
             <string>Codec:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="codecComboBox">
            <item>
             <property name="text">
              <string>MP3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Raw (16-bit)</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacerCodecSampleRate">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelSampleRate">
            <property name="text">
             <string>Sample Rate:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sampleRateComboBox">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <item>
             <property name="text">
              <string>11025</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>22050</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>44100</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacerSampleRateChannels">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelChannels">
            <property name="text">
             <string>Channels:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="channelsComboBox">
            <item>
             <property name="text">
              <string>Same as source</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Stereo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mono</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="algorithmQualityGroup">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Algorithmic Quality</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayoutAlgorithmQuality">
           <item>
            <widget class="QSlider" name="algorithmicQualitySlider">
             <property name="maximum">
              <number>9</number>
             </property>
             <property name="value">
              <number>5</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayoutAlgorithmLabels">
             <item>
              <widget class="QLabel" name="labelAlgorithmHighQuality">
               <property name="text">
                <string>0 (Higher quality, slower encode)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelAlgorithmLowQuality">
               <property name="text">
                <string>(Lower quality, faster encode) 9</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="vbrQualityGroup">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>VBR Quality</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayoutVbrQuality">
           <item>
            <widget class="QSlider" name="vbrQualitySlider">
             <property name="maximum">
              <number>9</number>
             </property>
             <property name="value">
              <number>4</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBelow</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayoutVbrLabels">
             <item>
              <widget class="QLabel" name="labelVbrHighQuality">
               <property name="text">
                <string>0 (Higher quality, larger file)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelVbrLowQuality">
               <property name="text">
                <string>(Lower quality, smaller file) 9</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="footerLine">
       <item>
        <widget class="QLabel" name="labelWebsite">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;a href=&quot;http://www.mjbshaw.com&quot; style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;www.mjbshaw.com&lt;/a&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextBrowserInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveAsButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save SWF As...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
